homeassistant/components/traccar/device_tracker.py

```diff
@@ -172,11 +159,24 @@ def __init__(self, api, hass, async_see, scan_interval,
             for custom_attr in self._custom_attributes:
                 if device_info.get(custom_attr) is not None:
                     attr[custom_attr] = device_info[custom_attr]
+                    if custom_attr in self._skip_accuracy_on:
+                        skip_accuracy_filter = True
+
+            accuracy = float(0)
+            if device_info.get('accuracy') is not None:
+                accuracy = device_info['accuracy']
+            if (not skip_accuracy_filter and self._max_accuracy > 0 and
+                    accuracy > self._max_accuracy):
+                _LOGGER.info('Excluded position by accuracy filter: %f (%s)',
```

Comment:
This would be "spammy", change it to debug or consider removing it.

Reply:
OK, Changed to debug.